main.py

```diff
@@ -30,123 +61,232 @@ def SEARCH():
 
     done=False #loop variable
 
+    howto = False #Instruction Menu
+    start = False #Game Started
+
+    #Initialise fonts
+    pygame.font.init()
+
     clickedbox=0 #to set default value
-    while not done:
+
+    #Initial Event
+    event = pygame.event.poll()
+
+    while event.type != pygame.QUIT:
+
+        windowSurface.blit(pygame.transform.scale(BG,(size)),(0,0))
+
+        event = pygame.event.poll()
+
+        if event.type == pygame.MOUSEMOTION:
+            x, y = event.pos
+
+        if start == False and howto == False:
+            if event.type == pygame.MOUSEBUTTONDOWN and event.button == 1:
+                xpress, ypress = event.pos
+                if howtoRect.collidepoint(event.pos):
+                    howto = True
+
+                elif startRect.collidepoint(event.pos):
+                    start = True
+                    break
+
+            title = pygame.font.SysFont("symbola",100)
+            textSurf1, textRect1 = text_objects("Find The Key!", title)
+            textRect1.center = ( (width/2), (height/2) - 100)
+
+            findkey = pygame.font.SysFont("liberationserif",20)
+            textSurf2, textRect2 = text_objects("LET'S FIND THE KEY!", findkey)
+            textRect2.center = ( (width/2), (height/2) + 150)
+
+            howtotext = pygame.font.SysFont("liberationserif",20)
+            textSurf3, textRect3 = text_objects("HOW-TO-PLAY", howtotext)
+            textRect3.center = ( (width/2), (height/2) + 225)
+            
+            windowSurface.blit(pygame.transform.scale(STARTGOLDKEY,(200,100)),(412,70))
+
+            if x>=392 and x<=632 and y>=509 and y<=559:

```

Comment:
If there is a better way to generalize this 'button hovering' code instead of mentioning the absolute mouse position, it would be better isn't it?

Reply:
With that being said, please find a way if there exists one, otherwise I shall reconsider this temporary fix